webpackShims/angular.js

```diff
@@ -1,7 +1,8 @@
 require('jquery');
 require('node_modules/angular/angular');
+require('node_modules/angular-translate/dist/angular-translate.min');
```

Comment:
no need to override the default import. We minify the distributable and prefer non-minified code for development.